src/controllers/commentController.js

```diff
@@ -0,0 +1,97 @@
+import commentServices from "../services/commentServices";
+import commentValidator from "../validation/commentValidation";
+import Util from "../utilities/util";
+import db from "../models";
+
+const util = new Util();
+
+const commentController = {
+  /**
+   * allows a user to make a comment
+   * @param {object} req - request object
+   * @param {object} res - response object
+   * @returns {object} success message | error
+   */
+  comment: async (req, res) => {
+    const { id } = req.decoded.user;
```

Comment:
Users should be able to **comment under a country** you are not referencing any country here. Comments have to reference the countries using appropriate associations.